lib/kwalify_to_json_schema/converter.rb

```diff
@@ -99,7 +99,13 @@ def process(target, kelem, path = [])
         target["type"] = "array"
         sequence = kelem["sequence"]
         if sequence.is_a? Array
-          process(target["items"] = {}, sequence.first)
+          rule = sequence.first
```

Comment:
I confirmed that Kwalify enforces that the sequence rule must exist so `rule` cannot be null here if the Kwalify schema itself is valid.